modules/dxp/apps/osb/osb-faro/osb-faro-contacts-demo/src/main/java/com/liferay/osb/faro/contacts/demo/internal/ContactsDemo.java

```diff
@@ -67,10 +90,10 @@ protected void activate() {
 				if (StringUtil.equals(
```

Comment:
Hey @MrJohn1911, I think now this if/else is useless since you are calling the same method. Please remove that.